CMake/platforms/miyoo_mini.cmake

```diff
@@ -0,0 +1,28 @@
+set(BUILD_ASSETS_MPQ OFF)

```

Comment:
For the assets, you can just have them go straight to the target directory, like we do for the original Xbox:

https://github.com/diasurgical/devilutionX/blob/6097a8a90a8580723078802bdcebfbffb073ca5b/CMake/platforms/xbox_nxdk.cmake#L10-L11

Reply:
Yes, I've already found out. That is quite a handy configuration but since I've also added packaging for MiniUI, which is basically the same OS/Device but with a different Frontend and folder structure, I copy the assets to two different places when packaging. Thank you very much!